lib/validation/postgres_config_validator.rb

```diff
@@ -8,6 +8,8 @@ def initialize(version)
         @config_schema = Validation::PostgresConfigValidatorSchema::PG_17_CONFIG_SCHEMA
       when "16"
         @config_schema = Validation::PostgresConfigValidatorSchema::PG_16_CONFIG_SCHEMA
+      when "18"
```

Comment:
Seems fine, but you may want to consider a hash-lookup approach instead of a case approach to reduce duplication.